tgui/packages/tgui/interfaces/MedicalRecords.js

```diff
@@ -314,59 +374,134 @@ const MedicalRecordsViewMedical = (_properties, context) => {
 const MedicalRecordsViruses = (_properties, context) => {
   const { act, data } = useBackend(context);
   const { virus } = data;
-  virus.sort((a, b) => (a.name > b.name ? 1 : -1));
-  return virus.map((vir, i) => (
-    <Fragment key={i}>
-      <Button
-        icon="flask"
-        content={vir.name}
-        mb="0.5rem"
-        onClick={() => act('vir', { vir: vir.D })}
-      />
-      <br />
-    </Fragment>
-  ));
+  const [searchText, setSearchText] = useLocalState(context, 'searchText', '');
+  const [sortId, _setSortId] = useLocalState(context, 'sortId', 'name');
+  const [sortOrder, _setSortOrder] = useLocalState(context, 'sortOrder', true);
+  return(
+  <Flex direction="column" height="100%">
+    <Flex>
+      <FlexItem grow="1" ml="0.5rem">
+        <Input
+          placeholder="Search by Name, Max Stages, or Severity"
+          width="100%"
+          onInput={(e, value) => setSearchText(value)}
+        />
+      </FlexItem>
+    </Flex>
+    <Section flexGrow="1" mt="0.5rem">
+      <Table className="MedicalRecords__list">
+        <Table.Row bold>
+          <SortButton id="name">Name</SortButton>
+          <SortButton id="max_stages">Max Stages</SortButton>
+          <SortButton id="severity">Severity</SortButton>
+        </Table.Row>
+        {virus
+          .filter(
+            createSearch(searchText, (vir) => {
+              return (
+                vir.name +
+                '|' +
+                vir.max_stages +
+                '|' +
+                vir.severity
+              );
+            })
+          )
+          .sort((a, b) => {
+            const i = sortOrder ? 1 : -1;
+            return a[sortId].localeCompare(b[sortId]) * i;
+          })
+          .map((vir) => (
+            <Table.Row
+              key={vir.id}
+              className={
+                'MedicalRecords__listVirus--' + vir.severity
+              }
+              onClick={() => act('vir', { vir: vir.D })}
+            >
+              <Table.Cell>
+                <Icon name="virus" /> {vir.name}
+              </Table.Cell>
+              <Table.Cell>{vir.max_stages}</Table.Cell>
+              <Table.Cell
+                color = {severities[vir.severity]}
+              >
```

Comment:
```suggestion
              <Table.Cell color={severities[vir.severity]}>
```
since you only have one attr, might as well put it on one line